nmosquery/common/routes.py

```diff
@@ -85,7 +85,12 @@ def __subscriptions_post(self):
             elif data["secure"] is False:
                 abort(400, "All subscriptions must be secure when operating with HTTPS")
         obj, created = self.query.post_ws_subscribers(data)
-        return (201 if created else 200, obj)
+        response = make_response(jsonify(obj), 201 if created else 200)
+        response.autocorrect_location_header = False
+        response.headers["Location"] = "/x-nmos/query/{}/subscriptions/{}".format(
```

Comment:
*potential knit-pick warning* - the Location Header only really serves a purpose for the 201 response code, not sure if it is misleading having it for a 200 response.. but probz doesn't matter.

Reply:
Agreed. This is just what the spec says to do, and does ensure some form of consistency for clients if they don't know a subscription already exists.